dm/config/task.go

```diff
@@ -714,8 +714,16 @@ func (c *TaskConfig) SubTaskConfigs(sources map[string]DBConfig) ([]*SubTaskConf
 		cfg.HeartbeatReportInterval = c.HeartbeatReportInterval
 		cfg.Meta = inst.Meta
 
-		cfg.From = dbCfg
-		cfg.To = *c.TargetDB
+		fromClone := dbCfg.Clone()
+		if fromClone == nil {
```

Comment:
maybe check (dbCfg!=nil) before clone looks a little more reasonable

Reply:
these checking are placed before pointer dereference, in case of `Clone` has a bug.